spec/requests/admin/creator_settings_spec.rb

```diff
@@ -55,7 +59,8 @@
         expect(creator.saw_onboarding).to eq(true)
         expect(creator.checked_code_of_conduct).to eq(true)
         expect(creator.checked_terms_and_conditions).to eq(true)
-        expect(response).to have_http_status(:ok)
+        # we redirect immadiately on success
+        expect(response).to have_http_status(:found)
```

Comment:
You can have a test case that also explains itself.
```suggestion
expect(response).to redirect_to(:root).and have_http_status(:found)
```